pom.xml

```diff
@@ -805,6 +815,19 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
```

Comment:
Should these be in the parent module scope? Seems like they're primarily used by child modules.